backend/app/ingest.py

```
@@ -888,3 +891,177 @@ async def get_available_documents(admin: dict = Depends(auth.require_admin)):
     job_ids = database.get_available_documents()
     return {"job_ids": job_ids}
+
+
+# =============================================================================
+# DOCUMENT DEFAULTS USER-TYPE OVERRIDE ENDPOINTS (Admin)
+# =============================================================================
+
+@router.get("/admin/documents/defaults/user-type/{user_type_id}", response_model=DocumentDefaultsUserTypeResponse)
+async def get_document_defaults_for_user_type(
+    user_type_id: int,
+    admin: dict = Depends(auth.require_admin)
```

Ruff (0.14.14)

[warning] Unused function argument: `admin`

(ARG001)